nimsuggest/nimsuggest.nim

```diff
@@ -166,12 +166,6 @@ proc execute(cmd: IdeCmd, file, dirtyfile: AbsoluteFile, line, col: int;
         ", dirtyFile: " & dirtyfile.string &
         "[" & $line & ":" & $col & "]")
   conf.ideCmd = cmd
-  if cmd == ideChk:
```

Comment:
``executeEpc`` calls ``execute``, not the oher way round but ``execute`` needs to have these hooks set. I think this refactoring is wrong.